### PR TITLE
Remove references to obsolete engine versions

### PR DESCRIPTION
**Pull request by thaJeztah**
There's probably more to address, but pushing what I had so far

### CODE REVIEW

**Comment by thaJeztah** [comment]
@usha-mandya ptal

File: storage/bind-mounts.md [hunk 1]
```diff
@@ -25,11 +25,8 @@ manage bind mounts.
 
 ## Choose the -v or --mount flag
 
-Originally, the `-v` or `--volume` flag was used for standalone containers and
-the `--mount` flag was used for swarm services. However, starting with Docker
-17.06, you can also use `--mount` with standalone containers. In general,
-`--mount` is more explicit and verbose. The biggest difference is that the `-v`
-syntax combines all the options together in one field, while the `--mount`
+In general, `--mount` is more explicit and verbose. he biggest difference is that
```
Comment (usha-mandya) [inline storage/bind-mounts.md:28]:
```
The biggest difference
```

File: engine/security/certificates.md [hunk 2]
```diff
@@ -23,12 +23,12 @@ A custom certificate is configured by creating a directory under
...
-> **Note**:
-> As of Docker 1.13, on Linux any root certificates authorities are merged
-> with the system defaults, including as the host's root CA set. On prior
-versions of Docker, and on Docker Enterprise Edition for Windows Server,
-> the system default certificates are only used when no custom root certificates
-> are configured.
+> **Note**
+>
+> On Linux any root certificates authorities are merged with the system defaults,
+> including as the host's root CA set. On Docker Enterprise Edition for Windows
```
Comment (usha-mandya) [inline engine/security/certificates.md:29]: Should we also remove the Docker Enterprise Edition here?

Comment (thaJeztah) [reply engine/security/certificates.md:29]: Ah, yes, I was looking at that. It was a bit tricky as it applies to either "Docker on Windows" or "Docker Desktop on Windows running "Windows containers". Let me have another look

**Comment by thaJeztah** [comment]
@usha-mandya updated; PTAL

**Review by usha-mandya — APPROVED** [review]
LGTM.

**Pull request merged**